app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="user-info-container">
+    <div class="post">
+        <h2>Post: <%= post.title %></h2>
```

Comment:
- [ ] The post title should be a link to post#show page. Please 🙏🏽 add this function to the post title to be able to see all comments.